libs/blocks/modal/modal.js

```diff
@@ -152,9 +152,11 @@ export async function getModal(details, custom) {
   const focusVisible = { focusVisible: true };
   const focusablesOnLoad = [...dialog.querySelectorAll(FOCUSABLES)];
   const titleOnLoad = dialog.querySelector('h1, h2, h3, h4, h5');
+  const isVideoFragment = dialog.querySelector('.fragment:first-child')?.dataset?.path?.includes("modals/videos") || false;
```

Comment:
No need for `|| false` in here. [includes() will return false if the string is not found](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes).

Reply:
Includes returns a boolean value but this has been added to handle the case where the querySelector does not find a fragment child, in such a scenario the value would be null and since null is not equal to false i have added "||false".

Reply:
Still no need for `|| false`. When using the [logical not operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_NOT) in `!isVideoFragment`, it will equal `true` if `isVideoFragment` has any falsy value, that includes the possible `null` from the querySelector not finding it's target.

Reply:
yes, i could use this alternate syntax here.
since i have used the !isVideoFragment inside the "if condition" on line 159 i had added that to improve the understandability of the code.
now that i have stated my train of thought for adding  "||false" do you think its valid or is it understandable without it as well?

Reply:
I think it's readable without it as well. Better keep the code in the least verbose version IMO

Reply:
okay, i have updated the code.